.github/workflows/application.yaml

```diff
@@ -28,42 +28,39 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        config:
-          {% for env in matrix_envs -%}
-          - environment: @{{ env.name }}
-            account_id: @{{ env.aws_account_id }}
-          {% endfor %}
+        {% for env in matrix_envs -%}
```

Comment:
not sure anymore but we had a discussion and one of the two (having or not this "config:" preserved the order of execution. More a question to myself.

Reply:
In fact, not sure it works ? I tried it here: https://github.com/tx-pts-dai/lacework-settings/actions/runs/9412855571/workflow and I got "A sequence was not expected"

Reply:
The thing is I did a PR that I had to rebase few times because of major changes in the repo main branch. Yesterday afternoon Demetrio merged the jinja changes which broke a few things. One of them is the matrix. Then I had to solve conflicts I was not sure it was working nor tested before so I kept the sytax. But I have the same doubts as you 🤷‍♂️